docs/modules/string.md

```diff
@@ -0,0 +1,329 @@
+# 🔠 String
+
+**`#bs.string:help`**
+
+Manage strings (text), allowing easy transformation and searching within the text.
+
+```{admonition} Technical Limitation
+:class: warning
+
+Due to the way string are handled, we don't support using `"` you need to use the `\"` instead.
+
+```
+
+---
+
+## 🔧 Functions
+
+You can find below all functions available in this module.
+
+---
+
+### Concat
+
+```{function} #bs.string:concat
+
+Merge multiple strings from the input list into a single string.
+
+:Inputs:
+  **Storage `bs:in string.concat.list`**: {nbt}`string` List of strings to merge
+
+:Outputs:
+  **Storage `bs:out string.concat`**: {nbt}`string` The merged string
+```
+
+_Merge `Hello ` and `World`:_
+
+```mcfunction
+data modify storage bs:in string.concat.list set value ["Hello ","World"]
+function #bs.string:concat
+tellraw @a [{"text":"The merged string is \""},{"storage":"bs:out","nbt":"string.concat"},{"text":"\""}]
+```
+
+> **Credits**: Aure31
+
+---
+
+### Convert Case
+
+:::::{tab-set}
+::::{tab-item} Uppercase
+
+```{function} #bs.string:upper
+
+Convert text to uppercase.
+
+:Inputs:
+  **Storage `bs:in string.upper.str`**: {nbt}`string` The string to convert to uppercase.
+
+:Outputs:
+  **Storage `bs:out string.upper`**: {nbt}`string` The uppercase string.
+```
+
+_Convert `hello world` to uppercase:_
```

Comment:
The example must help the user to understand the feature, here you are not showing the expecting result so we cannot understand it if we don't know what "uppercase" means.
I suggest to modify this line by "Convert `hello world` to its uppercase version `HELLO WORLD`:" or something like this.